vic/drivers/cesm/bld/vic.options.txt

```diff
@@ -0,0 +1,229 @@
+#######################################################################
+# VIC Model Options
+#######################################################################
+
+#######################################################################
+# Simulation Parameters
+#######################################################################
+NLAYER      3   # number of soil layers
+NODES       10  # number of soil thermal nodes
+TIME_STEP   3   # model time step in hours (set to 24 if FULL_ENERGY = FALSE, set to < 24 if FULL_ENERGY = TRUE)
+SNOW_STEP   3   # time step in hours for which to solve the snow model (should = TIME_STEP if TIME_STEP < 24)
+
+#######################################################################
+# Simulation Parameters Namelist
+#######################################################################
+# CONSTANTS     (put the parameter namelist path here) # Update default model paramters.  Format is "PARAM VAL".
+
+#######################################################################
+# Energy Balance Parameters
+#######################################################################
+FULL_ENERGY     FALSE   # TRUE = calculate full energy balance; FALSE = compute water balance only.  Default = FALSE.
+#CLOSE_ENERGY   FALSE   # TRUE = all energy balance calculations (canopy air, canopy snow, ground snow,
```

Comment:
Thanks for catching this.  I had removed this file but forgot to do it using the `git rm` so it didn't make it into the commit. The file `vic.globalconfig.txt` is the clean version we want in RASM.